featurebyte/persistent/base.py

```diff
@@ -178,7 +178,7 @@ async def find(
         page: int
             Page number for pagination
         page_size: int
-            Page size (0 to return all records)
+            Page size (0 to return all records, -1 to return iterator without count)
```

Comment:
Would it be clear if we introduce another function (type handling would be easier).